simtools/simtel/simtel_light_emission.py

```diff
@@ -0,0 +1,552 @@
+        command += str(self._simtel_source_path.joinpath("sim_telarray/LightEmission/"))
+        command += f"/{self.le_application[0]}"
+
+        if self.light_source_type == "led":
+            if self.le_application[1] == "variable":
+                command += f" -x {self.default_le_config['x_pos']['default'].to(u.cm).value}"
+                command += f" -y {self.default_le_config['y_pos']['default'].to(u.cm).value}"
+                command += f" -z {self.default_le_config['z_pos']['default'].to(u.cm).value}"
+                command += (
+                    f" -d {','.join(map(str, self.default_le_config['direction']['default']))}"
+                )
+                command += f" -n {self._calibration_model.get_parameter_value('photons_per_run')}"
+
+            elif self.le_application[1] == "layout":
+                x_origin = (
+                    self.default_le_config["x_pos_ILLN-01"]["default"]
+                    - self.default_le_config["x_pos"]["real"]
+                )
+                y_origin = (
+                    self.default_le_config["y_pos_ILLN-01"]["default"]
+                    - self.default_le_config["y_pos"]["real"]
+                )
+                z_origin = (
+                    self.default_le_config["z_pos_ILLN-01"]["default"]
+                    - self.default_le_config["z_pos"]["real"]
+                )
+                # light_source coordinates relative to telescope
+                command += f" -x {x_origin.to(u.cm).value}"
+                command += f" -y {y_origin.to(u.cm).value}"
+                command += f" -z {z_origin.to(u.cm).value}"
+                pointing_vector = self.calibration_pointing_direction()[0]
+                command += f" -d {','.join(map(str, pointing_vector))}"
+
+                command += f" -n {self._calibration_model.get_parameter_value('photons_per_run')}"
+
+                # same wavelength as for laser
+                command += f" -s {self._calibration_model.get_parameter_value('laser_wavelength')}"
+
+                # pulse
+                command += (
+                    f" -p Gauss:{self._calibration_model.get_parameter_value('led_pulse_sigtime')}"
+                )
+                command += " -a isotropic"  # angular distribution
+                # {self._calibration_model.get_parameter_value('led_pulse_offset')}"
+                # TODO further parameters require modification of application
+
+                # command += f" -s {self._calibration_model.get_parameter_value('led_var_photons')}"
+                # command += f" -s {self._calibration_model.get_parameter_value('pedestal_events')}"
+            command += f" -A {self.output_directory}/model/"
+            command += f"{self._telescope_model.get_parameter_value('atmospheric_profile')}"
+        elif self.light_source_type == "laser":
+            # TODO: this option requires the atmospheric profiles in the include directory,
```

Comment:
You mean to copy them in the include directory in the simtel_array include directory? I believe the better way would be to change the application to accept a path. Otherwise this is difficult to keep track of if and what file was actually used in the end.

Reply:
No, simtools would pull the profiles from the DB for each run, so this way we can be sure to use the right ones.